### ISSUE TITLE
update code

### ISSUE THREAD

**Issue by johndavid8** [gpsguru/traccar-ui-php#6]
can you please update you code where when the user/admin login can use all pages of traccar page. in your code its only show the map , how we can link to device and reports and geofence, etc? can you please update it. thank you

another thing do you have the traccar in php and mysql? its too difficult to edit it in javascrip.

thanks again

**Comment by jaimzj** [comment]
Yes, this is just a a very basic UI example for reference. With time I shall update it for sure.

As for PHP and MySQL . there is already another project of mine you may check, it has PHP integration of traccar.

**Comment by conde000011100** [comment]
Hi @jaimzj , how can we see your new project?
I'm very interested in the project

Best regards

**Comment by jaimzj** [comment]
You can check the other repositories of mine in github,.

**Comment by conde000011100** [comment]
Hi @jaimzj , Yodiyil is not yet available for demo?

I would like to see the platform in operation as I can create an account?

regards

**Comment by jaimzj** [comment]
I do not understand what you are referring to.

Whatever is available here on this github repository under user "gpsguru" is open and available to all.

**Comment by conde000011100** [comment]
your site https://yodiyil.com/

**Comment by jaimzj** [comment]
That has nothing to do with this repo !

this is  https://github.com/gpsguru/  and www.gpsguru.in   maintained repo's

**Comment by conde000011100** [comment]
Hello, I can send you an email for possible front end business of traccar...
